backends-velox/src/test/scala/org/apache/gluten/execution/VeloxAggregateFunctionsSuite.scala

```diff
@@ -50,6 +50,32 @@ abstract class VeloxAggregateFunctionsSuite extends VeloxWholeStageTransformerSu
       .set("spark.sql.sources.useV1SourceList", "avro")
   }
 
+  test("drop redundant partial sort which has pre-project when offload sortAgg") {
```

Comment:
Would you like to move this test case down, to the bottom of this file, probably? Thanks.